python/src/spark_rapids_ml/classification.py

```diff
@@ -979,9 +979,33 @@ def _logistic_regression_fit(
             if self.getStandardization() is True and is_sparse is True:
                 concated = concated.toarray()
 
+            num_pad_zero_cols = 0
+            if self.getStandardization() is True and concated.shape[1] % 32 != 0:
+                num_pad_zero_cols = concated.shape[1] % 32
```

Comment:
Shouldn't the number of padding columns be 32 - concated.shape[1] % 32 ?

Reply:
Good catch!